### PR TITLE
[12.0][IMP] Improvement in hotel reservation module

### PR DESCRIPTION
**Pull request by vimalpatelserpentcs**

### CODE REVIEW

File: hotel_reservation/__manifest__.py [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-        'report/hotel_reservation_report_template.xml',
-        'report/report_view.xml',
-        'views/assets.xml',
-        'wizards/hotel_reservation_wizard.xml',
+    "name": "Hotel Reservation Management",
+    "version": "12.0.1.0.0",
+    "author": "Odoo Community Association (OCA), Serpent Consulting \
+                Services Pvt. Ltd., Odoo S.A.",
+    "category": "Generic Modules/Hotel Reservation",
+    "license": "AGPL-3",
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/__manifest__.py:9]: Add a website parameter please

File: hotel_reservation/models/hotel_reservation.py [hunk 2]
```diff
@@ -112,70 +113,117 @@ class HotelReservation(models.Model):
...
+    )
+    folio_id = fields.Many2many(
+        "hotel.folio",
+        "hotel_folio_reservation_rel",
+        "order_id",
+        "invoice_id",
+        string="Folio",
+    )
+    no_of_folio = fields.Integer("No. Folio", compute="_compute_folio_id")
+    dummy = fields.Datetime("Dummy")
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/models/hotel_reservation.py]: Any idea why this field was added?

File: hotel_reservation/models/hotel_reservation.py [hunk 3]
```diff
@@ -195,99 +243,113 @@ def unlink(self):
...
-            if reserv_rec.state != 'draft':
-                raise ValidationError(_('You cannot delete Reservation in %s\
-                                         state.') % (reserv_rec.state))
+            if reserv_rec.state != "draft":
+                raise ValidationError(
+                    _(
+                        "You cannot delete Reservation in %s\
+                                         state."
+                    )
+                    % (reserv_rec.state)
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/models/hotel_reservation.py]: Simplify this : "Sorry, you can only delete the reservation when it's draft!"

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: hotel_reservation/__manifest__.py (hunk 1 above)
Comment (vimalpatelserpentcs) [reply hotel_reservation/__manifest__.py:9]: Done

File: hotel_reservation/models/hotel_reservation.py (hunk 2 above)
Comment (vimalpatelserpentcs) [reply hotel_reservation/models/hotel_reservation.py]: Actually, there is no major use of these fields but only use in the calendar view for display, but now we have removed it and in a calendar view we have added check-in and  check-out dates

File: hotel_reservation/models/hotel_reservation.py (hunk 3 above)
Comment (vimalpatelserpentcs) [reply hotel_reservation/models/hotel_reservation.py]: Done

**Review by JayVora-SerpentCS — APPROVED** [review]

**Pull request merged**